Small changes

thimotedupuch opened the pull request:
This is my first contribution to this project.

- I started by performing a global formatting with `Runic.jl`, on a separate commit so reviewers should be able to see the diffs correctly.

- I added the missing docs for `intersectLineToPlane3D`
 
- I implemented the CPU version of `radialDistortion!`

Commit:
Formatting via Runic.jl

Commit:
improved docs for intersectLineToPlane3D + getindex refactoring

Commit:
CPU imlementation for radialDistortion! with @tturbo

Commit:
add LoopVectorization to the project's dependencies

Affie commented:
Hi @thimotedupuch, thanks for the contribution. I'm not familiar enough with that part of the code to help review. But I do see the tests are failing because of code that moved from Manifolds.jl to LieGroups.jl. Give me a few days and I'll upgrade to LieGroups.jl.

Affie committed:
Merge branch 'develop' into small_fixes

Affie commented:
Hi @thimotedupuch. I fixed the LieGroups.jl issue and also fixed a config issue with code coverage.
Would you mind adding a test for `radialDistortion!`?

Commit:
Refactor + test for radialDistortion!

thimotedupuch commented:
So, this took longer than expected: I took the time to review the repo as a whole and noticed many inconsistencies. You can close this PR or merge it if you prefer. I plan to initiate more significant changes to the code. I intend to update the dependencies, remove some, and simplify many parts of the codebase.
Most importantly, we need to clarify which other packages interface with this one.
In any case, I’ve rewritten the radialDistortion! function and added a basic test for it.

Affie commented:
Contributions to CameraModels.jl are very welcome, and we would especially appreciate someone willing to take on a more dedicated maintenance role. Please note that the package is used by RoME.jl and Caesar.jl, so changes should follow semantic versioning with clear deprecations where applicable. Beyond that, feel free to move things forward.
